From: Thorsten Glaser <tg@debian.org>
Date: Tue, 27 Mar 2012 20:16:13 +0000
Subject: Use syscall _newselect instead of select on ppc64

Same as 0008-Use-syscall-_newselect-instead-of-select-on-ppc.diff
as #394928 applies to the kernel, not the userspace, and these are
but chroots on the same kernel.

Index: pkg-dietlibc/ppc64/select.S
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ pkg-dietlibc/ppc64/select.S	2012-03-27 19:53:25.000000000 +0000
@@ -0,0 +1,3 @@
+#include "syscalls.h"
+
+syscall(_newselect,select)
